services/QuillLMS/client/app/bundles/PremiumHub/containers/AccountManagementTemp.tsx

```diff
@@ -0,0 +1,655 @@
+import * as React from 'react';
```

Comment:
This is annoying for review, but the bulk of this file is just a duplicate of `AccountManagement`.